cmd/wallet-web/tailwind.js

```diff
@@ -492,8 +493,62 @@ module.exports = {
           },
         },
       };
+      const credentialPreview = {
```

Comment:
All these styles seem to be easy to add in tailwind. Are we planning to reuse them elsewhere? In https://github.com/trustbloc/wallet/pull/973/files#diff-f19fc0e3ca858adf3c716fbee60f6d382dacb66ef836c0df223130b6ff564b23R24-R41 it is not duplicated

Reply:
we still have credential details pages to be implemented.  Most likely the same stuff can be used there if not will create a separate to add in tailwind

Reply:
If we'd need this credential preview component elsewhere, we should make it reusable and put it under `components` folder as well